src/common/packager.ts

```diff
@@ -162,6 +162,19 @@ export class Packager {
 
                 let spawnOptions = { env: reactEnv };
 
+                // Since expo@37, you must specify the sourceExts parameter so that the packager can load additional files, such as custom fonts:
```

Comment:
```suggestion
                // Since Expo 37, you must specify the sourceExts parameter so that the packager can load additional files, such as custom fonts:
```